src/components/RatingsChart/RatingsChart.tsx

```diff
@@ -255,6 +283,15 @@ export class RatingsChart extends React.PureComponent<RatingsChartProperties, an
         this.rating_graph = this.svg.append('g')
             .attr('transform', 'translate(' + margin.left + ',' + margin.top + ')');
 
+        /* Win-loss pie chart goes to the right of the rating graph... */
```

Comment:
Lets avoid trailing ellipsis in comments in general, they indicate omitted text, doesn't seem applicable.